panda/src/particlesystem/particleSystem.I

```diff
@@ -59,12 +59,12 @@ soft_start(PN_stdfloat br) {
 }
 
 /**
- * Causes system to use birth rate set by set_birth_rate()
+ * Causes system to use birth rate set by set_birth_rate(), with the timer for the system's initial birth offset by the value of first_birth_offset_time
```

Comment:
Oof - I should have mentioned that we really like docstrings to be word-wrapped to a ruler of 78 characters, for readability. ;)

Reply:
Right, that's done, I believe! ^_^